src/MSBuild.Abstractions/MSBuild.Abstractions.csproj

```diff
@@ -2,6 +2,7 @@
 
   <PropertyGroup>
     <TargetFramework>netcoreapp3.1</TargetFramework>
+    <Nullable>enable</Nullable>
```

Comment:
Can you add thi once to the `Directory.Build.props` file instead of duplicating this property in every project file?